app/components/views/Offset/index.tsx

```diff
@@ -226,20 +231,30 @@ export const Offset = (props: Props) => {
     setBeneficiaryAddress(address);
   };
 
+  const getApprovalValue = (): string => {
+    const costAsNumber = Number(cost);
+    const costPlusOnePercent = costAsNumber + costAsNumber * APPROVAL_SLIPPAGE;
+    return costPlusOnePercent.toFixed(18); // ethers does not like more than 18 digits and returns with "underflow" otherwise
```

Comment:
this depends on the token, I believe USDC approval would still throw underflow at 18 decimals, since its a 6 decimal token

Reply:
Thanks! Good to know.

I pushed the fix